flags.go

```diff
@@ -80,6 +80,7 @@ func defineFlagVar() *FlagVar {
 	flag.IntVar(&flagVar.listenerHTTPPortLocalMapping, "listener-http-local-mapping", defaultListenerPort,
 		"Port that is mapped to HTTP port of the local k3d cluster using --port 9080:80@loadbalancer when "+
 			"creating the KCP cluster")
+	flag.StringVar(&flagVar.skrWatcherImage, "skr-watcher-image", "", "Image of the SKR watcher")
```

Comment:
Please state that this will overwrite the default behaviour, also add what the default behaviour is, if this flag is not set